### DISCUSSION TITLE
Local video file with subtitles

### DISCUSSION THREAD

**Discussion by warrenkc** [FrostCo/AdvancedProfanityFilter#526] [Q&A]
Hi. I have a local video file that can play in VLC or other video players with subtitles. Does any one know how to use something like Advanced Profanity Filter on local non streaming websites? Thanks!

**Comment by richardfrost** [comment]
Thanks for the question @warrenkc. This is being discussed more in #518, but as of now there is not a supported way to do this with APF. There may be other projects out there that could do something similar, but I can't really speak to those. It is something that I'd like to support in some fashion in APF or a spin-off project.

**Reply by warrenkc** [reply]
Thank you as well. I was thinking about making a solution that works locally.
The application needs to process video and audio files, extract subtitles, identify profanity, and edit the audio accordingly. 

I just found a project on GitHub which can handle this: 
https://github.com/mmguero/cleanvid

I think it would be great if there was a GUI (user interface) for it.
